### ISSUE TITLE
[UX] Find a better name for 'Text formats'

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3709]
From https://github.com/backdrop/backdrop-issues/issues/3595#issuecomment-486387037

> We tried to remove the language `Text format` from Backdrop core because that phrase won't make sense to anyone who's not specifically worked with it in Drupal 7. The use of the phrase `Text format` on the node edit form specifically was something that presented issues in every single Usability study on Drupal 7.

> The kinds of people who are using the node edit form are the least likely people to have ever seen the crazy back-end label `Text format` before. The node edit form is a page intended for every person to use - not only =high-level admins who have access to the `Text format` configuration page, and understand our specific use of that phrase.

> When we shipped with Backdrop 1.0 the back-end configuration page and menu link was named `Text editors and formats`. 

> I somehow missed the opportunity to argue against that change that restored the phrase `Text format`, and was very sad when I saw the old, confusing, and Drupal-specific label reappear.

I'm open to better suggestions as to what to call this thing, but let's choose something that makes more sense than what we know didn't work for Drupal. 

Ideally, we'd also choose something that would make sense to laypeople as well as administrators so that the terminology can be consistent across both kinds of interfaces.

I'd like to dig deeper, and address the underlying problems with `Text editors and formats` since I believe that was a step in the right direction. 

Let's take another step forward, instead :)

Related (ordered newest to oldest): 
* https://github.com/backdrop/backdrop-issues/issues/3595
* https://github.com/backdrop/backdrop-issues/issues/2736
* https://github.com/backdrop/backdrop-issues/issues/1091
* https://github.com/backdrop/backdrop-issues/issues/1032

**Comment by Graham-72** [comment]
I agree 100%.

Here is a suggestion:

On the node edit form change 'Formatting options' to be 'Editor options'. And change the label 'Editor' to be 'Editor type'. Do not show any of this if the user has only one option.

On the administration form change 'Add text format' to be 'Add editor type'.
Within the configuration page change 'Roles allowed to use this text format' with 'Roles allowed to use this editor type'.

Retain the administration form name 'Text editors and formats' though perhaps it should be 'Editors and formats' since images get included.

It seems to me that setting an Editor type (Plain text) with Editor = none on the administration page is perfectly valid because the user is still able to edit their text. The user should not be shown this 'none' but only see the label, e.g. 'Plain text'.

**Comment by olafgrabienski** [comment]
I like "Editor type", much better than just "Editor". Before I read your suggestion, I was also thinking about how to complement "Editor" and had the idea for "Editor mode". I've the impression that "Editor mode" is even more clearly not a editor role or person, what do you native English speakers think about it?

**Comment by olafgrabienski** [comment]
> Retain ... 'Text editors and formats' though perhaps it should be 'Editors and formats' ...

Why not "Editor types" at this place?

**Comment by BWPanda** [comment]
Do other CMS's have a similar feature? If so, what terminology do they use?

**Comment by klonos** [comment]
I have no useful suggestions at the moment, but I agree that we should change this to something that makes sense to content editors (that do not necessarily have any experience with Drupal/Backdrop, hence no familiarity with our lingo).

**Comment by jenlampton** [comment]
I have updated the the parent issue with a lengthy explanation of exactly what these things do, maybe that will help us come up with a better name for them?

**Comment by BWPanda** [comment]
From what I can see, Wordpress uses the term 'Editor':

![image (1)](https://user-images.githubusercontent.com/2385329/56762816-68638f80-67a9-11e9-97d2-c9dfa16920b9.png)
![image (2)](https://user-images.githubusercontent.com/2385329/56762819-6ac5e980-67a9-11e9-9f29-ed915494ed3c.png)

**Comment by klonos** [comment]
Thanks @BWPanda 👍 

Just to clarify, I believe that "Classic" vs "Block" in the context of WP refers to the pre-Gutenberg vs Gutenberg editors.

**Comment by jenlampton** [comment]
Do WordPress `Editors` include Filters too? (is there any PHP processing on the output before things are rendered, or only changing things on input?)

**Comment by BWPanda** [comment]
> Just to clarify, I believe that "Classic" vs "Block" in the context of WP refers to the pre-Gutenberg vs Gutenberg editors.

Yes, but I think this is the closest they have to allowing the end-user to select their own thingamabobber (from what I could see). Drupal/Backdrop might be unique in this regard...

@jenlampton It seems they have some sort of filtering in code, not sure if there's anything in the UI though: https://core.trac.wordpress.org/browser/tags/5.1.1/src/wp-includes/formatting.php#L4071

**Comment by stpaultim** [comment]
Originally posted this here: https://github.com/backdrop/backdrop-issues/issues/1519

I agree that naming is probably the problem here. I login as admin and go to create a page. I see that the default editor is `filtered html` and I have a WYSIWYG text editor that provides me with push button tools to generate that html. 

I'm thinking that I would like more options in my WYSIWYG editor and I see that there is a `full html` editor available, that will probably provide me with more WYSIWYG options to generate that html. I click on that and all my WYSIWYG options go away. Now, I'm thinking WTF? 

;-)

![Create_Page___sprinting](https://user-images.githubusercontent.com/3144571/62426607-9dd31f80-b6ac-11e9-999b-548f81a6ced5.png)

![Create_Page___sprinting](https://user-images.githubusercontent.com/3144571/62426620-c1966580-b6ac-11e9-819b-e9287613496a.png)

The problem is that I am thinking about the editor options in terms of features that they will provide me, NOT in terms of the filters they employ and neither the naming or help text provided is very useful in making that distinction. 

If we were to keep both, I would name them something like (names that help end user understand their purpose):

* Limited WYSIWYG
(help text: filtered HTML)
* Full HTML Filter
(help text: direct edit your HTML)

**Comment by stpaultim** [comment]
OK, I'm a little confused now, because this issue seems to be out-of-date. As you can see in the screenshots above, we are currently using the term `editor.` Is that a recent change and should this issue be closed and another opened to discuss the naming of specific editors?

**Comment by klonos** [comment]
@stpaultim unless I'm mistaken, I believe that we are discussing the UI terminology used in `/admin/config/content/formats` (and the respective menus etc.). So basically aiming to have this bit of the UI make more sense:

<img width="966" alt="Screen Shot 2019-08-04 at 9 47 28 pm" src="https://user-images.githubusercontent.com/2423362/62427831-b7528680-b701-11e9-8282-fb9624d51203.png">
